<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="OpenTemplateImageButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Template Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenSourceImageButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Source Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayTemplateImageLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DisplaySourceImageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>平方差匹配CV_TM_SQDIFF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标准平方差匹配CV_TM_SQDIFF_NORMED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>相关匹配CV_TM_CCORR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标准相关匹配CV_TM_CCORR_NORMED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>相关匹配CV_TM_CCOEFF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标准相关匹配CV_TM_CCOEFF_NORMED</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
